src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -109,6 +111,10 @@ const MapConfig = t.intersection([
       zoomLevel: t.number,
     }),
     baseLayer: t.keyof(baseLayers),
+    mouseMode: t.keyof({
+      default: null,
+      magnification: null,
+    }),
```

Comment:
I would move this to the `t.partial` section below so that existing map visualizations don't break (this makes the config setting optional).